packages/editor-ui/src/components/mixins/globalLinkActions.ts

```diff
@@ -39,7 +39,6 @@
 
 			const actionAttribute = clickedElement.getAttribute('data-action');
 			if(actionAttribute && typeof this.availableActions[actionAttribute] === 'function') {
-				e.preventDefault();
```

Comment:
did you test this change with every action it's used?

Reply:
with using `<i18n>` tag for translation this is not needed anymore, rolled back all changes in this file